content/efcore/getting-started/samples/generating-databases-and-data-models/generating-from-code-example/README.md

```diff
@@ -0,0 +1,9 @@
+# Generating from Code
+
+This code example shows what the database schema would be for the model included in the example.
+
+## Code Files:
+- [Book.cs](Book.cs)
+- [Program.cs](Program.cs)
+- [Reader.cs](Reader.cs)
+- [LibraryContext.cs](LibraryContext.cs)
```

Comment:
Remove extra newline (line 10) from this file. It's causing an error in the code example service.